common/djangoapps/student/views/management.py

```diff
@@ -743,6 +744,18 @@ def create_account_with_params(request, params):
     else:
         compose_and_send_activation_email(user, profile, registration)
 
+    # If flag is enabled - creates record for user with approved status of the photo verification
+    if configuration_helpers.get_value(
+            'AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION',
+            settings.FEATURES.get('AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION', False)
+    ):
+        SoftwareSecurePhotoVerification.objects.create(
+            status="approved",
+            user_id=user.id,
+            submitted_at=datetime.datetime.now(UTC),
```

Comment:
That's weird that the field is not auto-filled but I suppose that it's a case of logic.